.github/workflows/release.yml

```diff
@@ -11,10 +14,48 @@ jobs:
         matrix:
             os: [ubuntu-latest, windows-latest, macos-latest]
     steps:
-      - name: "Create filename"
-        id: fn
+      - name: "Create base filename for all artifacts"
+        id: basefn
         shell: bash
-        run: echo "FILENAME=build/rctab_${{ github.ref_name }}_${{ runner.os }}.zip" >> $GITHUB_OUTPUT
+        run: |
+          FILEPATH=$(echo rctab_${{ github.ref_name }}_${{ runner.os }} | sed -e 's/\//_/g')
+          echo "FILEPATH=$FILEPATH" >> $GITHUB_OUTPUT
+
+      # Normalize platform-specific filepaths generated by gradle
+      - name: "Create .zip filename"
+        id: zipfn
+        shell: bash
+        run: echo "FILEPATH=build/${{ steps.basefn.outputs.FILEPATH }}.zip" >> $GITHUB_OUTPUT
+      - name: "Get extension"
+        id: ext
+        shell: bash
+        run: |
+          if [ ${{ runner.os }} == 'Windows' ]; then
+            echo "EXT=.exe" >> $GITHUB_OUTPUT
+          elif [ ${{ runner.os }} == 'Linux' ]; then
+            echo "EXT=.deb" >> $GITHUB_OUTPUT
+          else
+            echo "EXT=.dmg" >> $GITHUB_OUTPUT
+          fi
+      - name: "Get jpackage output filepath"
+        id: jpackagefn
+        shell: bash
+        run: |
+          # TODO Sync version number with Main.java and build.gradle (github.com/BrightSpots/rcv/issues/662)
+          # The version numbers are hardcoded because the files below include the version number in them,
+          # and while we could use some regex to figure out the version number automatically, it seems cleaner
+          # to know the expected version number upfront.
+          if [ ${{ runner.os }} == 'Windows' ]; then
```

Comment:
I noticed the file I downloaded inside `Package.zip` is called `rctab_feature_issue-57_standalone-executable_Windows.exe`, which seems off... it should be the version number, shouldn't it?

Reply:
It will be in the final release! Just not in development builds, since it uses the ref name. The ref name is the branch name for these dev builds, and the release name for release builds.